.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  release:
+    if: github.event.repository.fork == false
+
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: step-security/harden-runner@v1
+        with:
+          allowed-endpoints:
+            api.github.com:443
+            github.com:443
+
+      - name: Get version from tag
+        id: tag_name
+        run: echo ::set-output name=current_version::${GITHUB_REF#refs/tags/v}
+        shell: bash
+
+      - uses: actions/checkout@v2
+
+      - uses: mindsers/changelog-reader-action@v2
+        id: changelog_reader
+        with:
+          version: ${{ steps.tag_name.outputs.current_version }}
+
+      - name: Get common links from changelog
+        id: changelog
+        run: |
+          # Parse the changelog for common links
+          _links="$(grep -Pz -o '(?s)<!-- common links -->.*' ${GITHUB_WORKSPACE}/CHANGELOG.md)"
+          _links="${_links//'%'/'%25'}"
+          _links="${_links//$'\n'/'%0A'}"
+          _links="${_links//$'\r'/'%0D'}"
+          # Set output 'links' to $_links
```

Comment:
i wonder if it might be simpler to just grab *every* link from the file, sort | uniq, and dump them at the bottom of the release body?

Reply:
Adjusted to pull all links. Updated test release at https://github.com/xt0rted/eslint-plugin-react/releases/tag/v7.30.0. Looks good to me so far.